helm-charts/ingress-controller/templates/Deployment.yaml

```diff
@@ -23,10 +22,11 @@ spec:
         - -c
         - |-
           echo name: $EG_NAME > /opt/eg-config/config.yaml &&
-          cat /opt/eg-config/eg-secondary.yaml >> /opt/eg-config/config.yaml &&
+          cat /opt/eg-config/easegress-server.yaml >> /opt/eg-config/config.yaml &&
```

Comment:
It need not to generate `config.yml`, if you write server configuration with `name` configuration.